tests/integration/test_dimension.py

```diff
@@ -47,7 +58,206 @@ def type_fixture(self, request):
         return dimension_dicts, expected_types
 
 
+class DescribeIntegratedNewDimension(object):
+    """Integration-test suite for `cr.cube.dimension.NewDimension` object.
+
+    This is a temporary object, roughly a shim, to provide the new Dimension interface
+    to FrozenSlice. Once that work is completed, this will become DescribeDimension and
+    the NewDimension object will be merged into the Dimension object.
+    """
+
+    def it_constructs_from_a_legacy_Dimension_object(self, legacy_dimension):
+        dimension = NewDimension(legacy_dimension, None)
+        assert isinstance(dimension, NewDimension)
+
+    def it_provides_access_to_all_elements_in_its_collection(self, legacy_dimension):
+        legacy_dimension = CrunchCube(CR.ECON_BLAME_WITH_HS).dimensions[0]
```

Comment:
No need to override fixture here

Reply:
Oh, right, good catch, must have been left from before I added the fixture, thanks :)